### DISCUSSION TITLE
Add a ore dictionary to Deepslate

### DISCUSSION THREAD

**Discussion by Omgise** [Roadhog360/Et-Futurum-Requiem#257] [Improvements]
Its ore dictionary name should be the same as stone

**Comment by Roadhog360** [comment]
Does stone have an OreDictionary name by default?

**Reply by Omgise** [reply]
> Does stone have an OreDictionary name by default?

Yes, it has an OreDictionary name
![2023-04-09_13 19 46](https://user-images.githubusercontent.com/106000018/230755896-feb1d210-6fc8-4f7b-8e74-fde3e0ef9dca.png)

**Comment by GregoriusT** [comment]
Oh right "stone" and "cobblestone" are the OreDicts used for ANY type of Stone or Cobblestone, so that Mods like Underground Biomes and similar can have their own variants of Rock without screwing up the Recipes.

Question is, does vanilla allow Cobbled Deepslate to be used in Crafting Recipes for things like Pistons and Furnaces, and regular Deepslate for things like Stone Pressure Plates and Buttons?

**Comment by Roadhog360** [comment]
I revisited this, I will not add Deepslate to the dictionary for stone because deepslate is not a stone substitute in vanilla.
